### PR TITLE
docs: add example for: awscc_efs_access_point

### PR DESCRIPTION
**Pull request by ishwar-amzn**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates OR Closes #0000

### CODE REVIEW

File: examples/resources/awscc_efs_access_point/awscc_efs_access_point.tf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+resource "awscc_efs_access_point" "this" {
+  file_system_id = awscc_efs_file_system.this.id
```
Comment (rahul-amzn) [inline examples/resources/awscc_efs_access_point/awscc_efs_access_point.tf:2]: Add `awscc_efs_file_system` resource

File: examples/resources/awscc_efs_access_point/awscc_efs_access_point_posix_user.tf [hunk 2]
```diff
@@ -0,0 +1,18 @@
+resource "awscc_efs_access_point" "this" {
+  file_system_id = "fs-0dbbfe225fc860016" #awscc_efs_file_system.this.id
```
Comment (rahul-amzn) [inline examples/resources/awscc_efs_access_point/awscc_efs_access_point_posix_user.tf:2]: remove comment and add `awscc_efs_file_system` resource block

**Review by rahul-amzn — CHANGES_REQUESTED** [review]

File: examples/resources/awscc_efs_access_point/awscc_efs_access_point_posix_user.tf (hunk 2 above)
Comment (ishwar-amzn) [reply examples/resources/awscc_efs_access_point/awscc_efs_access_point_posix_user.tf]: Thanks @rahul-amzn , Added awscc_efs_file_system resource and dynamic reference.

File: examples/resources/awscc_efs_access_point/awscc_efs_access_point_posix_user.tf [hunk 3]
```diff
@@ -0,0 +1,32 @@
+resource "awscc_efs_access_point" "this" {
+  file_system_id = awscc_efs_file_system.this.id
+  posix_user = {
+    gid = 0
+    uid = 0
```
Comment (wellsiau-aws) [inline examples/resources/awscc_efs_access_point/awscc_efs_access_point_posix_user.tf:5]: Ideally let's not use root for this

can you change the gid and uid to something higher than 1000? 

i.e. 1001

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
minor changes to the posix user uid

**Comment by wellsiau-aws** [comment]
@ishwar-amzn can you make the suggested change for the gid and uid? thanks!

**Comment by ishwar-amzn** [comment]
Hi @wellsiau-aws,
As suggested, updated posix user uid

**Review by wellsiau-aws — APPROVED** [review]
LGTM, thank you @ishwar-amzn

**Comment by marcosentino** [comment]
@ishwar-amzn Thank you for the contribution!

**Pull request merged**